src/Equinox.Cosmos/Equinox.Cosmos.fsproj

```diff
@@ -20,13 +24,13 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="FsCodec" Version="2.0.1" />
```

Comment:
Neither Equinox or Equinox.Core reference FsCodec. Once FsCodec.STJ is created as a package, then it can just reference that. Or am I misunderstanding?

Reply:
Wow. Doh; at least you're awake!
(and the L33 point is that dev depends like minver and infra like FSharp.Core go in a separate cluster before real depensd)
(It def was an intentional thing to not have Equinox or Core depend on the codec; I just clean forget that decoupling had been accomplished and not relinquished!)